### PR TITLE
Restoring UI

### PR DESCRIPTION
**Pull request by AhmedKorim**
Some changes weren't merged, they are added to this PR

1. Deposite/Mixer UI
2. Theme switching

The Mixer isn't initialized

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,18 +29,23 @@ Webb web app with mixer features.
...
 {
   "resolutions": {
+    "@webb-tools/api": "file:../webb.js/packages/api/build",
+    "@webb-tools/types": "file:../webb.js/packages/types/build",
+    "@webb-tools/app-util": "file:../webb.js/packages/app-util/build",
+    "@webb-tools/sdk-core": "file:../webb.js/packages/sdk-core/build",
     "@webb-tools/sdk-mixer": "file:../webb.js/packages/sdk-mixer/build",
-
-
+    "@webb-tools/mixer-client": "^0.0.6"
```
Comment (shekohex) [inline README.md:43]: Update the version:

```suggestion
    "@webb-tools/mixer-client": "^0.0.7"
```

**Review by shekohex — COMMENTED** [review]
Looks good, just a small changes and this is ready to go!

**Review by shekohex — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -11,12 +11,12 @@
   },
   "resolutions": {
     "@polkadot/api": "^3.4",
-    "@webb-tools/api": "^0.1.2-31",
-    "@webb-tools/types": "^0.1.2-31",
-    "@webb-tools/app-util": "^0.1.2-31",
-    "@webb-tools/sdk-core": "^0.1.2-31",
-    "@webb-tools/sdk-mixer": "^0.1.2-31",
-    "@webb-tools/mixer-client": "^0.0.6"
+	"@webb-tools/api": "^0.1.2-33",
```
Comment (drewstone) [inline package.json:14]: Spacing is off

File: packages/page-mixer/src/index.tsx [hunk 3]
```diff
@@ -21,63 +20,39 @@ const subMenu = [
...
+      </Col>
+      <Col span={24}>
+        <DepositConsole />
       </Col>
     </Row>
   );
 };
 
-export default PageMixer;
+export default PageSwap;
```
Comment (drewstone) [inline packages/page-mixer/src/index.tsx:58]: Mixer should be used instead of Swap

File: packages/page-mixer/src/index.tsx [hunk 4]
```diff
@@ -21,63 +20,39 @@ const subMenu = [
...
-        variant={'warning'}
-        content={"Mixer isn't supported on  selected chain "}
-        title='Unsupported Feature'
-      />
-    );
-  }
-
-  const { changeTabs: changeSubMenu, currentTab: currentSubMenu } = useTabs<MixerPageType>('deposit');
-  const { init } = useMixer();
+const PageSwap: FC = () => {
```
Comment (drewstone) [inline packages/page-mixer/src/index.tsx:23]: Mixer should be used instead of Swap

File: packages/page-mixer/src/index.tsx [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
-import { Col, Information, Row, useTabs } from '@webb-dapp/ui-components';
-import { Asset } from '@webb-tools/sdk-mixer';
-import React, { FC, useEffect } from 'react';
 
 import { DepositConsole } from './components/deposit';
-import { WithdrawConsole } from './components/withdraw';
+import { LiquidityInformation } from './components/common';
 
-type MixerPageType = 'deposit' | 'withdraw';
+type SwapTabType = 'deposit' | 'withdraw';
```
Comment (drewstone) [inline packages/page-mixer/src/index.tsx:10]: Mixer instead of Swap

**Review by drewstone — CHANGES_REQUESTED** [review]
Mixer language should be used instead of Swap, and some styling should be fixed (including line breaks / linting that are complaining. Looks good otherwise!

**Comment by AhmedKorim** [comment]
@drewstone  I thought this is merged already, You can merge it now to have the app running.
I will add more fixes on functionality and structure on the next PR

**Pull request merged**